sdk/communication/Azure.Communication.CallingServer/src/autorest.md

```diff
@@ -1,60 +1,65 @@
 # Azure.Communication.CallingServer
 
 When a new version of the swagger needs to be updated:
+
 1. Open Azure.Communication.sln and under sdk\communication, run `dotnet msbuild /t:GenerateCode` to generate code.
 
 2. Upload the Azure.Communication.Call.dll to the [apiview.dev tool](https://apiview.dev/).
-If any of the new objects needs to be overwritten, add the required changes to the 'Models' folder.
+   If any of the new objects needs to be overwritten, add the required changes to the 'Models' folder.
```

Comment:
Why do we have many tab and spacing changes here. Can you make sure just the url is updated and nothing else is modified?